src/main/docker/Dockerfile.centos

```diff
@@ -88,7 +88,13 @@ RUN     env && \
            --update '$connector' --value \$\{TOMCAT_PORT_SSL\} \
         ${CATALINA_HOME}/conf/server.xml && \
         rpm --erase xmlstarlet && \
-        #
+
+	# Configure logging.properties
+        echo "1catalina.org.apache.juli.AsyncFileHandler.maxDays = \${TOMCAT_LOGGING_MAXDAYS}" >>${CATALINA_HOME}/conf/logging.properties && \
```

Comment:
If those `&&` are to stop on failure, know that you can do the same with `bash -e`.